packages/language-server/package.json

```diff
@@ -26,6 +26,7 @@
   "main": "./dist/index.js",
   "typings": "dist/src/index",
   "dependencies": {
+    "checkpoint-client": "1.1.2",
```

Comment:
We add this just to read the signature?
What is the overhead in packages installed/added by adding this dependency?

Reply:
yes, it's only for the signature in both the server and the vscode extension

Reply:
adds about 10 additional packages as can be seen in this diff: https://github.com/prisma/language-tools/pull/320/commits/eacc5b681a188ae527213b4f49996c7ccf7026be

Reply:
I will suggest we should keep checkpoint here as if we change something regarding fingerprint in checkpoint, it vscode will also receive that change.

Reply:
Never indicated otherwise @pantharshit00.